include/PatternClip.h

```diff
@@ -23,17 +23,17 @@
  */
  
 
-#ifndef BB_CLIP_H
-#define BB_CLIP_H
+#ifndef PATTERN_CLIP_H
+#define PATTERN_CLIP_H
 
 #include "ClipView.h"
 
 
-class BBClip : public Clip
+class PatternClip : public Clip
```

Comment:
A diagram here would be cool, so that you know that this is inside the song editor, not in the PatternStore.